analyzer/uncategorized/addresses.go

```diff
@@ -42,6 +45,15 @@ func StringifyEthAddress(ethAddr []byte) (apiTypes.Address, error) {
 	return StringifyOcAddress(ocAddr)
 }
 
+func EthAddrReference(preimage string) *string {
```

Comment:
```suggestion
func EthAddrReference(eth_addr_hex string) *string {
```
to make it clearer what the fn does.